sdk/keyvault/azure-keyvault-administration/tests/conftest.py

```diff
@@ -28,8 +28,7 @@ def add_sanitizers(test_proxy):
     azure_managedhsm_url = azure_managedhsm_url.rstrip("/")
     azure_attestation_uri = os.environ.get("AZURE_KEYVAULT_ATTESTATION_URL","https://fakeattestation.azurewebsites.net")
     azure_attestation_uri = azure_attestation_uri.rstrip('/')
-    storage_name = os.environ.get("BLOB_STORAGE_ACCOUNT_NAME", "blob_storage_account_name")
-    storage_endpoint_suffix = os.environ.get("KEYVAULT_STORAGE_ENDPOINT_SUFFIX", "keyvault_endpoint_suffix")
+    storage_url = os.environ.get("BLOB_STORAGE_URL", "https://storagename.blob.core.windows.net")
```

Comment:
Can we use something like https://fakename.blob.core.windows.net which is more obvious?

Reply:
Technically it'll be recorded as `https://Sanitized.blob.core.windows.net`, so I can update this to align.

Reply:
My concern is secret scanner may report it finds "secret" in our checked source code. :)